test/rules/no-mocha-global-arrows.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var RuleTester = require('eslint').RuleTester,
+    rules = require('../../').rules,
+    ruleTester = new RuleTester(),
+    expectedErrorMessage = 'Do not pass arrow functions to it()';
+
+ruleTester.run('no-mocha-global-arrows', rules['no-mocha-global-arrows'], {
+
+    valid: [
+        'it()',
+        'it(function() { assert(something, false); })',
+        'it("should be false", function() { assert(something, false); })',
+        {
+            // In this example, `it` is not a global.
```

Comment:
Mocha also has an [`require`](http://mochajs.org/#require) interface in which you would require mocha functions. In this case they would not be globals so I wonder if we should support this interface too?
What do you think?